plugins/nodes/src/js/pages/nodes/nodes-grid/NodesGridContainer.js

```diff
@@ -21,9 +22,8 @@ class NodesGridContainer extends mixin(StoreMixin, QueryParamsMixin) {
     this.state = {
       filteredNodes: [],
       filters: {health: 'all', name: '', service: null},
-      hasLoadingError: false,
+      mesosXHRError: null,
       hiddenServices: [],
-      mesosStateErrorCount: 0,
```

Comment:
Fair enough, I just don't know if removing this without implementing it elsewhere is the best approach. What do you think?